packages/api-client/src/api/index.ts

```diff
@@ -1,2 +1,4 @@
 export * from './Api';
 export * from './functions';
+
+export { ApiClientFactory } from './ApiClientFactory';
```

Comment:
I couldn't get `export *` in this nested submodule to expose ApiClientFactory, even when it's not default.  I'd welcome changes, but I'm happy with this approach